libautoscoper/src/Tracker.cpp

```diff
@@ -321,6 +321,9 @@ void Tracker::optimize(int frame, int dFrame, int repeats, int opt_method, unsig
   // 4) Run Downhill Simplex (Nelder-Mead) on the best guess from the PSO
   // 5) Update the volume pose based on the best guess
 
+  // Set the optimization parameters
+  this->latest_optimization_parameters_ = OptimizationParameters(repeats, dFrame, opt_method, max_iter, min_limit, max_limit, cf_model, max_stall_iter, trial_.guess);
```

Comment:
Since we are not "setting" the parameters used for the optimization but only keeping track of them.

I then suggest to move this call at the end of the function just after:

```
    std::cerr << "Tracker::optimize(): Frame " << trial_.frame
         << " done in " << totalIter << " total iterations" << std::endl;
```

and changing the comment `Set the optimization parameters`  to `Log optimization parameters`